neo/Ledger/MemoryPool.cs

```diff
@@ -165,6 +171,36 @@ public IEnumerator<Transaction> GetEnumerator()
 
         IEnumerator IEnumerable.GetEnumerator() => GetEnumerator();
 
+        public BigInteger GetSenderFee(UInt160 sender)
+        {
+            _txRwLock.EnterReadLock();
+            try
+            {
```

Comment:
Remove try?

Reply:
Yes try is not necessary, I will correct it.